nekoyume/Assets/_Scripts/UI/Widget/WorldMap.cs

```diff
@@ -220,9 +221,14 @@ public void SetWorldInformation(WorldInformation worldInformation)
                 }
 
                 var buttonWorldId = worldButton.Id;
+                var unlockRow = TableSheets.Instance.WorldUnlockSheet
+                    .Select(pair => pair.Value)
+                    .FirstOrDefault(row => row.WorldIdToUnlock == buttonWorldId);
```

Comment:
You can use `OrderedList`.🙂

```suggestion
                var unlockRow = TableSheets.Instance.WorldUnlockSheet
                    .OrderedList
                    .FirstOrDefault(row => row.WorldIdToUnlock == buttonWorldId);
```